### PR TITLE
Sync development with master

### PR DESCRIPTION
**Pull request by JGiter**
This needs to unblock https://github.com/energywebfoundation/iam-cache-server/pull/292

### CODE REVIEW

File: src/modules/claim/claim.controller.ts [hunk 1]
```diff
@@ -238,7 +238,7 @@ export class ClaimController {
       'should return claims for namespaces like ' +
       '`admin.roles.myApp.apps.myOrg.iam.ewc`',
   })
-  public async getByParentNamespace(@Param('id') id: string) {
+  public async getByParentNamespace(@Param('namespace') id: string) {
```
Comment (jrhender) [inline src/modules/claim/claim.controller.ts:241]: Is `namespace` the correct value?

**Review by jrhender — APPROVED** [review]
Approved with one question

File: src/modules/claim/claim.controller.ts (hunk 1 above)
Comment (JGiter) [reply src/modules/claim/claim.controller.ts:241]: If I am not mistaken it is local identifier, which is not part of API. `namespace` is also field of the `RoleClaim`, so it is looks consistent

Comment (JGiter) [reply src/modules/claim/claim.controller.ts:241]: Although there is still some bug with this endpoint, as it is always returns [ ] though locally everything works fine for `sync_dev_with_master` branch

**Pull request merged**